package.json

```diff
@@ -10,6 +10,7 @@
     "aws-sdk": "^2.879.0",
     "body-parser": "^1.18.3",
     "bull": "^3.20.1",
```

Comment:
will using `bull` and `bullmq` at the same time cause any problems?

Reply:
🤔 .... i would think so ... i haven't found anything saying they will not.  also, bull is just putting jobs in redis.